webapp/src/components/chat/chat-list/dialogs/DeleteChatDialog.tsx

```diff
@@ -0,0 +1,62 @@
+import { Button } from '@fluentui/react-button';
+import { Tooltip, makeStyles } from '@fluentui/react-components';
+import {
+    Dialog,
+    DialogActions,
+    DialogBody,
+    DialogContent,
+    DialogSurface,
+    DialogTitle,
+    DialogTrigger,
+} from '@fluentui/react-dialog';
+import { Delete16 } from '../../../shared/BundledIcons';
+
+const useClasses = makeStyles({
+    root: {
+        width: '450px',
+    },
+    actions: {
+        paddingTop: '10%',
+    },
+});
+
+interface IEditChatNameProps {
+    chatId: string;
+    chatName: string;
+}
+
+export const DeleteChatDialog: React.FC<IEditChatNameProps> = ({ chatName }) => {
+    const classes = useClasses();
+
+    return (
+        <Dialog modalType="alert">
+            <DialogTrigger>
+                <Tooltip content={'Delete chat session'} relationship="label">
+                    <Button icon={<Delete16 />} appearance="transparent" aria-label="Edit" />
+                </Tooltip>
+            </DialogTrigger>
+            <DialogSurface className={classes.root}>
+                <DialogBody>
+                    <DialogTitle>Are you sure you want to delete chat {chatName}?</DialogTitle>
+                    <DialogContent>
+                        This will permanently delete the chat for you—but not for Chat Copilot. You need to delete
+                        anything that you have shared (files, tasks, etc.) separately.
```

Comment:
Is this true?

Reply:
Yeah, we'll basically be gravetombing the chat so it's never accessible from the front end, but all the artifacts will be maintained in storage (unless we decide to change it later) 

I also added a TODO to get the right copy from Matthew to communicate this better when we enable deletion